<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogForm</class>
 <widget class="QWidget" name="LogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="chbStartLog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>启动日志功能</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labLogFrqkey">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>记录频率: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbbLogFrq">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>2秒</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5秒</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10秒</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30秒</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1分钟</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5分钟</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10分钟</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30分钟</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1小时</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbBaseInfo">
     <property name="title">
      <string>基础信息</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="chbBaseUserName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbBaseUserId">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>用户ID</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbBaseSystemName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>系统名</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbBaseHostName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>主机名</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbBaseSystemRelease">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>内核版本</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbBaseSystemVersion">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>系统版本</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbBaseMachineType">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>机器类型</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbProcessInfo">
     <property name="title">
      <string>进程信息</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="chbProcessPid">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>PID</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbProcessUser">
          <property name="text">
           <string>用户</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="chbProcessName">
          <property name="text">
           <string>进程名</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbProcessPRI">
          <property name="text">
           <string>PRI</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="chbProcessState">
          <property name="text">
           <string>状态 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbProcessNice">
          <property name="text">
           <string>Nice</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="chbProcessMemory">
          <property name="text">
           <string>占用内存</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbProcessCmdLine">
          <property name="text">
           <string>命令行</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QCheckBox" name="chbProcessMemoryRate">
          <property name="text">
           <string>内存占用率</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbProcessCPURate">
          <property name="text">
           <string>CPU占用率</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>195</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbCPUInfo">
     <property name="title">
      <string>CPU信息</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="chbCPUCount">
        <property name="text">
         <string>核数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbCPUType">
        <property name="text">
         <string>型号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ChbCPUUsage">
        <property name="text">
         <string>使用率</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>416</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbMemoryInfo">
     <property name="title">
      <string>内存信息</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="chbMemoryTotalRAM">
        <property name="text">
         <string>内存大小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMemoryFreeRAM">
        <property name="text">
         <string>可用内存</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMemoryRAMUsage">
        <property name="text">
         <string>内存使用率</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMemoryTotalSwap">
        <property name="text">
         <string>交换区大小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMemoryFreeSwap">
        <property name="text">
         <string>可用交换区</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMemorySwapUsage">
        <property name="text">
         <string>交换区使用率</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbNetworkInfo">
     <property name="title">
      <string>网络信息</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="labNetReceive">
          <property name="text">
           <string>下载</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labNetSend">
          <property name="text">
           <string>上传</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QCheckBox" name="chbNetRecvBytes">
          <property name="text">
           <string>总字节</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbNetSentBytes">
          <property name="text">
           <string>总字节</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QCheckBox" name="chbNetRecvRate">
          <property name="text">
           <string>速率</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbNetSentRate">
          <property name="text">
           <string>速率</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QCheckBox" name="chbNetRecvPackets">
          <property name="text">
           <string>总包数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbNetSentPackets">
          <property name="text">
           <string>总包数</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QCheckBox" name="chbNetRecvErrors">
          <property name="text">
           <string>错误</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbNetSentErrors">
          <property name="text">
           <string>错误</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QCheckBox" name="chbNetRecvErrorRate">
          <property name="text">
           <string>错误率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbNetSentErrorRate">
          <property name="text">
           <string>错误率</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QCheckBox" name="chbNetRecvDrop">
          <property name="text">
           <string>丢包</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbNetSentDrop">
          <property name="text">
           <string>丢包</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QCheckBox" name="chbNetRecvDropRate">
          <property name="text">
           <string>丢包率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbNetSentDropRate">
          <property name="text">
           <string>丢包率</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>74</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbFileSystemInfo">
     <property name="title">
      <string>文件系统信息</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QCheckBox" name="chbFSIsMounted">
          <property name="text">
           <string>是否挂载</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbFSBlockFree">
          <property name="text">
           <string>可用块数</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QCheckBox" name="chbFSDeviceName">
          <property name="text">
           <string>设备名</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbFSTotalBytes">
          <property name="text">
           <string>容量 </string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <widget class="QCheckBox" name="chbFSMountDir">
          <property name="text">
           <string>目录</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbFSFreeBytes">
          <property name="text">
           <string>可用大小</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <widget class="QCheckBox" name="chbFSMountType">
          <property name="text">
           <string>类型</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbFSUsage">
          <property name="text">
           <string>使用率</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <item>
         <widget class="QCheckBox" name="chbFSBlockSize">
          <property name="text">
           <string>块大小</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbFSLabel">
          <property name="text">
           <string>Label</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_20">
        <item>
         <widget class="QCheckBox" name="chbFSBlockCount">
          <property name="text">
           <string>总块数</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbFSUuid">
          <property name="text">
           <string>UUID</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
